.swiftformat

```diff
@@ -50,3 +50,4 @@
 --extensionacl on-declarations # extensionAccessControl
 --patternlet hoist # hoistPatternLet
 --indent 4 # indent
+--anonymousforeach ignore # preferForLoop
```

Comment:
not particularly fan of enforcing the usage of one or the other, II would not add any of these two rules.